runtime/vanilla/src/lib.rs

```diff
@@ -519,7 +519,7 @@ impl orml_tokens::Config for Runtime {
 }
 
 parameter_types! {
-    pub const GetNativeCurrencyId: CurrencyId = CurrencyId::Native;
+    pub const GetNativeCurrencyId: CurrencyId = CurrencyId::HKO;
```

Comment:
This should be something like `DEV`.

Reply:
@yz89 what do we want exactly?

Reply:
@yz89 do we really need a currency for vanilla?

Reply:
looks like acala doesn't have a currency for mandala https://github.com/AcalaNetwork/Acala/blob/master/primitives/src/currency.rs#L175-L193 @yz89 @kaichaosun

Reply:
Ok, I now think it make sense to use `HKO`. It shouldn't to take a place on CurrencyId.